lib/compress/zstd_compress.c

```diff
@@ -572,14 +564,14 @@ ZSTD_bounds ZSTD_cParam_getBounds(ZSTD_cParameter param)
         bounds.upperBound = 1;
         return bounds;
 
-    case ZSTD_c_splitBlocks:
+    case ZSTD_c_useBlockSplitter:
         bounds.lowerBound = 0;
         bounds.upperBound = 1;
         return bounds;
 
     case ZSTD_c_useRowMatchFinder:
-        bounds.lowerBound = (int)ZSTD_urm_auto;
-        bounds.upperBound = (int)ZSTD_urm_enableRowMatchFinder;
+        bounds.lowerBound = (int)ZSTD_ps_auto;
```

Comment:
minor : it would be a good place to `static_assert()` that the order of `enum` has not been messed with.